apollo-router/tests/integration/lifecycle.rs

```diff
@@ -225,30 +225,6 @@ async fn test_cli_config_preview() {
     );
 }
 
-#[tokio::test(flavor = "multi_thread")]
```

Comment:
Is this a test we should retain but use a different experimental feature for testing?